### ISSUE TITLE
Capacitor support

### ISSUE THREAD

**Issue by jjacobs33589** [kolbasa/cordova-plugin-apkupdater#25]
I've been using this plugin for my cordova app for a few months and it's been working great!

I'm in the process of converting from cordova to capacitor and am wondering if I can still use this?

**Comment by kolbasa** [comment]
Unfortunately, I have not yet had time to check what is needed for Capacitor.

However, according to the documentation, Cordova plugins can also be used with Capacitor without any problems.
https://capacitorjs.com/docs/v2/cordova/using-cordova-plugins

A warning beforehand: I have rewritten the complete plugin the last few days.
A lot has changed. A preview can be found here:

`v2.0.0`:
https://github.com/kolbasa/cordova-plugin-apkupdater/tree/refactoring

Demo-Apk:
https://github.com/kolbasa/cordova-plugin-apkupdater-demo

I am almost finished with the changes. 

Then I want to write at least a detailed tutorial how to use the plugin with Ionic > 2 / Capacitor.
I'm on it, but right now I can't tell you how to configure it there.

**Comment by kolbasa** [comment]
I have tried a bit with the latest Ionic version. Here is my first successful attempt. Of course, I have to say that I'm not really familiar with the new Ionic versions. But maybe it already helps.

```js
import {Platform} from '@ionic/angular';
import {Component} from '@angular/core';
import {HttpClient} from '@angular/common/http';

@Component({
  selector: 'app-home',
  templateUrl: 'home.page.html',
  styleUrls: ['home.page.scss'],
})

export class HomePage {

  endpoint = 'https://raw.githubusercontent.com/kolbasa/cordova-plugin-apkupdater-demo/master/update';

  constructor(private httpClient: HttpClient, public platform: Platform) {
    platform.ready().then(this.downloadAndInstall.bind(this)).catch(console.error);
  }

  async downloadAndInstall() {
    const apkUpdater = (window as any).ApkUpdater;

    const response = await this.httpClient.get<any>(this.endpoint + '/manifest.json').toPromise();

    const remoteVersion = response.version;
    const installedVersion = (await apkUpdater.getInstalledVersion()).version.name;

    if (remoteVersion > installedVersion) {
      await apkUpdater.download(this.endpoint + '/update.zip', {password: 'aDzEsCceP3BPO5jy', onDownloadProgress: console.log});
      await apkUpdater.install();
    }
  }

}

```

**Comment by kolbasa** [comment]
I have now tried Capacitor and implemented a TypeScript Api on Plugin-Version: `2.2.0`.

Installation:
```
npm install cordova-plugin-apkupdater
```

Sample Implementation:
https://github.com/kolbasa/cordova-plugin-apkupdater/wiki/Sample-Implementation

If you have problems with it, feel free to reopen the issue.
